### DISCUSSION TITLE
TextInput select all text when receives focus.

### DISCUSSION THREAD

**Discussion by NgYueHong** [filamentphp/filament#12407] [Help] [answered]
### Package

Form builder

### Package Version

3.2

### How can we help you?

How to make the TextInput field so that when it is on focus, it will select all the text in it?

**Comment by leandrocfe** [comment, marked as answer]
```php
->extraInputAttributes(['onClick' => 'this.select()'])
```
?

**Reply by NgYueHong** [reply]
It works!!!! Thank you very much for your help. :D
